pkg/userdata/helper/kubelet.go

```diff
@@ -131,6 +134,51 @@ func KubeletSystemdUnit(kubeletVersion, cloudProvider, hostname string, dnsIPs [
 	return b.String(), nil
 }
 
+// kubeletConfiguration returns marshaled kubelet.config.k8s.io/v1beta1 KubeletConfiguration
+func kubeletConfiguration(clusterDomain string, clusterDNS []net.IP) (string, error) {
+	var (
```

Comment:
Maybe you can use "k8s.io/utils/pointer" instead of having literal vars declaration :)

Reply:
done 👍